galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -78,6 +79,21 @@ class CollectionRemoteSerializer(LastSyncTaskMixin, pulp_viewsets.CollectionRemo
     last_sync_task = serializers.SerializerMethodField()
     created_at = serializers.DateTimeField(source='pulp_created', required=False)
     updated_at = serializers.DateTimeField(source='pulp_last_updated', required=False)
+    proxy_url = serializers.CharField(
```

Comment:
I just noticed that there is no validation on the URL. I think this used to be performed by the pulp serializer, but got overridden by this. Maybe use a URLField?

Reply:
@newswangerd pulp uses a CharField https://github.com/pulp/pulpcore/blob/master/pulpcore/app/serializers/repository.py#L93

the only url that is validated is `url` field https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/serializers.py#L136

Reply:
Not sure if URLField accepts user:pass and port.

```
URLField
A RegexField that validates the input against a URL matching pattern. Expects fully qualified URLs of the form http://<host>/<path>.

Corresponds to django.db.models.fields.URLField. Uses Django's django.core.validators.URLValidator for validation.

Signature: URLField(max_length=200, min_length=None, allow_blank=False)
```

Reply:
@rochacbruno  `proxy_url = serializers.URLField(...)` seems to work. It's just validating proxy_url on the serializer, not the transformed value that gets stored in the DB.